### PR TITLE
remove javax.annotation.Nonnull from files containing a different Non…

### PR DESCRIPTION
**Pull request by davidmotson**
…Null annotation

NO_RELEASE_CHANGE

### CODE REVIEW

**Comment by rlazo** [comment]
This is more widespread, as reported by running this command

```
$ rg javax.annotation.Nonnull -l |  awk -F'/' '{print $1}' |sort -u
firebase-common
firebase-config
firebase-firestore
firebase-inappmessaging
```

could you also fix those instances? thanks

**Comment by davidmotson** [comment]
done

**Comment by rlazo** [comment]
is this change still pending?

File: firebase-config/bandwagoner/src/main/java/com/googletest/firebase/remoteconfig/bandwagoner/RealtimeFragment.java [hunk 1]
```diff
@@ -94,7 +94,7 @@ private void toggleRealtime(View view, Boolean isChecked) {
         frc.addOnConfigUpdateListener(
             new ConfigUpdateListener() {
               @Override
-              public void onUpdate(ConfigUpdate configUpdate) {
```
Comment (rlazo) [inline firebase-config/bandwagoner/src/main/java/com/googletest/firebase/remoteconfig/bandwagoner/RealtimeFragment.java:97]: This is new, is this intended?

Comment (davidmotson) [reply firebase-config/bandwagoner/src/main/java/com/googletest/firebase/remoteconfig/bandwagoner/RealtimeFragment.java:97]: yes, in order to match the supertype.

**Review by rlazo — APPROVED** [review]

**Pull request merged**